### PR TITLE
docs: updated feed input documentation

### PR DESCRIPTION
**Pull request by jessmetivier**
## Description
Updated **Feed Input** docs.

### CODE REVIEW

File: stories/feed-input/feed-input.stories.js [hunk 1]
```diff
@@ -177,11 +190,11 @@ export const inputGrowth = () => `
                   aria-label="Feed" 
                   aria-required 
                   rows="4">With every line\n"textarea" grows upward\nuntil it reaches\nthe max height</textarea>
-        <button class="fd-button fd-feed-input__submit-button" 
+        <button class="fd-button sap-icon--feeder-arrow fd-feed-input__submit-button"
```
Comment (salarenko) [inline stories/feed-input/feed-input.stories.js:193]: To create a button with icon we compose it like that:
```
<button>
    <i class="sap-icon-class"></i>
</button>
```
There is no need to add `sap-icon--feeder-arrow` class to the `button` element.

**Review by salarenko — CHANGES_REQUESTED** [review]
LGTM, small issues:

- Should this component still have "IN DEVELOPMENT" label?
image

**Review by mikerodonnell89 — COMMENTED** [review]
Should be good after the button fix

File: stories/feed-input/feed-input.stories.js (hunk 1 above)
Comment (jessmetivier) [reply stories/feed-input/feed-input.stories.js:193]: Thanks Michal. I removed the "In Development" tag but fixing all the button code maybe needs another PR?

**Review by JKMarkowski — APPROVED** [review]
LGTM

**Pull request merged**